File: test/fixtures/output/node/request/cookies.js
var request = require("request");


var jar = request.jar();
jar.setCookie(request.cookie("foo=bar"), "http://mockbin.com/har");
jar.setCookie(request.cookie("bar=baz"), "http://mockbin.com/har");

request({
  "method": "POST",
  "url": "http://mockbin.com/har",
  "jar": jar
}, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: test/fixtures/output/java/nethttp/cookies.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://mockbin.com/har"))
    .header("cookie", "foo=bar; bar=baz")
    .method("POST", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());
